README.md

```diff
@@ -4,6 +4,25 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/8400b327ea3d328c9f5e/maintainability)](https://codeclimate.com/github/fterh/heimdall/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/8400b327ea3d328c9f5e/test_coverage)](https://codeclimate.com/github/fterh/heimdall/test_coverage)
 
+Features added:
```

Comment:
This is a remnants of how my ReadME changed. I can update the readme correctly if you would like to move forward.